src/Compilers/Core/Portable/DiagnosticAnalyzer/AsyncQueue.cs

```diff
@@ -52,7 +52,7 @@ public void Enqueue(TElement value)
         {
             if (!EnqueueCore(value))
             {
-                throw new InvalidOperationException($"Cannot call {nameof(Enqueue)} when the queue is already completed.");
+                throw new InvalidOperationException(string.Format(CodeAnalysisResources.CannotCallWhenQueueIsCompleted, nameof(Enqueue)));
```

Comment:
>string.Format [](start = 52, length = 13)

I believe calls to overloads of this API that do not take IFormatProvider are going to be flagged by WorldReady tools. We should be explicit about what culture is used to format the string, null is fine if we think this is what we want.